pg-index-health-spring-boot-demo/src/main/java/io/github/mfvanek/pg/index/health/demo/dto/ForeignKeyMigrationRequest.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2019-2024. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health-demo
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+package io.github.mfvanek.pg.index.health.demo.dto;
+
+import io.github.mfvanek.pg.connection.ConnectionCredentials;
+
+public record ForeignKeyMigrationRequest(
+    ConnectionCredentials credentials

```

Comment:
Sensitive data should not be passed as a request parameter. It is better to remove this class